packages/salesforcedx-vscode-apex-debugger/src/telemetry/telemetry.ts

```diff
@@ -52,8 +57,8 @@ export class TelemetryService {
     }
   }
 
-  private getEndHRTime(hrstart: [number, number]): string {
+  private getEndHRTime(hrstart: [number, number]): number {
```

Comment:
what does this change do?

Reply:
was this the root cause of the issue?

Reply:
`public sendTelemetryEvent(
    eventName: string,
    properties?: { [key: string]: string },
    measurements?: { [key: string]: number }
  ): void`

startupTime should be passed in via measurements, so that it can be viewed as a log metrics. So I need to change the function to return a number type. So measurements has a parameter like this:
`{
'startupTime': 1
}`